drivers/place/router.cr

```diff
@@ -0,0 +1,87 @@
+require "placeos-driver"
+
+class Place::Router < PlaceOS::Driver
+end
+
+require "./router/signal_graph"
+
+class Place::Router < PlaceOS::Driver
+  generic_name :Switcher
+  descriptive_name "Signal router"
+  description <<-DESC
+    A universal matrix switcher.
+    DESC
+
+  def on_load
+    on_update
+  end
+
+  def on_update
+    # TODO: load connections config
```

Comment:
I assume this is going to be utilising `from_json` and adding edges to the `@siggraph` in `Router::Core`?

Reply:
Yep correct.